### DISCUSSION TITLE
Is there a way to read the value of a YAML parameter defined in a Quarto qmd document from a Jupyter code block in Julia?

### DISCUSSION THREAD

**Discussion by plessl** [quarto-dev/quarto-cli#5159] [Q&A]
Dear all

I think this question has come up before but I can't find the answer in the archive.

Is there a way to read the value of a YAML parameter defined in a Quarto qmd document from a Jupyter code block in Julia?

e.g. 

````
---
status: draft
---
This is a Quarto document

```{julia}
 # <== how to access "status" here? 

```
````

My specific use case is that I would like to define a meta data field "status" declaring the maturity of a particular Quarto page and add a disclaimer to the rendered output depending on that status of the document, e.g. draft, revised version, release candidate, final. Thus it would be convenient to include the code for formatting the disclaimer from an include file and pass the value of status to this code. 

Cheers
 Christian

**Comment by mcanouil** [comment]
Short answer: no, there is not yet a way to do that.

This being said, depending on what you want to do I would suggest to use profiles and conditional contents, see <https://quarto.org/docs/projects/profiles.html> and <https://quarto.org/docs/authoring/conditional.html>.

Additionally, there is computations parameters, but I do not know and never tried to use it with Julia, see <https://quarto.org/docs/computations/parameters.html>.

**Comment by plessl** [comment]
Knitr parameters look interesting, but to my understanding I need to stick to juypter when working with Julia code cells.

Is passing data from the YAML metadata section to code cells executed with Juypter impossible by design? Or could this feature be added?

**Reply by mcanouil** [reply]
Jupyter and Knitr both work with parameters but with a slightly different syntax, see <https://quarto.org/docs/computations/parameters.html>.
